src/pymorize/rule.py

```diff
@@ -284,3 +286,64 @@ def global_attributes_set_on_rule(self):
         result = {attr: getattr(self, attr, None) for attr in attrs}
         result["creation_date"] = creation_date
         return result
+
+    def get_global_attributes(self, cmor_version):
+        """Return the global attributes for the rule"""
+        if not hasattr(self, "controlled_vocabularies"):
+            raise AttributeError(
+                "The rule does not have the controlled vocabularies attribute set."
+            )
+
+        # Template the global attributes mapping file
+        global_attributes_mapping_path = files("pymorize.data").joinpath(
+            f"global_attributes_mappings/{cmor_version}.yaml.j2"
+        )
+        with open(global_attributes_mapping_path) as file:
+            template = Template(file.read())
+
+        # (MA) TODO: move this to the rule -> rule.get_dict_from_attrs()
+        rule_dict = {
+            "controlled_vocabularies": self.controlled_vocabularies,
+            "data_request_variable": self.data_request_variable,
+            "source_id": self.source_id,
+            "model_component": self.model_component,
+        }
+
+        global_attributes = yaml.safe_load(template.render(rule_dict))
```

Comment:
Since you are rendering the template based upon the information expressed by attributes on the `Rule`, I would see if something like this works:

```suggestion
        global_attributes = yaml.safe_load(template.render(self.__dict__))
```

That way, you get all attributes and can use them in your template. For example:
```python
class Example:
    def __init__(self, x):
        self.x = x
    
    def method(self):
        return "I'm a method!"

obj = Example(42)
print(obj.__dict__)  # {'x': 42}
```

I'm not sure if you need to unpack it like this instead:
```suggestion
        global_attributes = yaml.safe_load(template.render(**self.__dict__))
```